src/SonarQube/SolutionBinding.sqconfig

```diff
@@ -1,4 +1,10 @@
 {
   "ServerUri": "https://dory.sonarsource.com/sonarqube/",
-  "ProjectKey": "sonarlint-visualstudio"
+  "ProjectKey": "sonarlint-visualstudio",
+  "Profiles": {
+    "CSharp": {
```

Comment:
Can be not use the server language key? Because this is going to be horrible to try and [correct/account for later](https://jira.sonarsource.com/browse/SVS-94).